cpu/stm32l0/include/periph_cpu.h

```diff
@@ -82,6 +82,17 @@ typedef struct {
     uint8_t ev_irqn;        /**< event IRQ */
 } i2c_conf_t;
 
+/**
+ * @brief   Define the number of available PM modes
+ */
+#define PM_NUM_MODES    (2U)
+
+/**
+ * @brief   Override the default initial PM blocker
+ * @todo   we block all modes per default, until PM is cleanly implemented
+ */
+#define PM_BLOCKER_INITIAL  { .val_u32 = 0x01010101 }
```

Comment:
Just asking, do you have a reference for this hardcoded value ?

Reply:
It is based on the periph_cpu.h file from stm32f1 family that seems to implement the low power mode feature too.

Reply:
The [default value is the same](https://github.com/RIOT-OS/RIOT/blob/master/sys/pm_layered/pm.c#L33), I'm wondering if it's really required here.

Reply:
What this means is that there are several reasons why we MUST block PM modes. The reasons are mostly peripheral related, e.g. we don't have a flexible/dynamic way on which we can disable/enable some peripherals which SHOULD block some power modes. For instance, if we allow to go to the deepest sleep mode (peripherals off), and we make use of UART, it won't work and we have undefined behaviour, since our current peripheral implementations are unaware if we're sleeping and how deep. To solve this problem, we block by default the power modes, so the device won't go to sleep unless we say so explicitly. Until we (re)implement this conditions for all our peripherals, we need to block them by default.

Reply:
> we need to block them by default.

I know, but this is already the case by default ,see [the link](https://github.com/RIOT-OS/RIOT/blob/master/sys/pm_layered/pm.c#L33) I gave